ampd/src/lib.rs

```diff
@@ -478,10 +478,8 @@ impl App {
                 handlers::stellar_verify_msg::Handler::new(
                     verifier.clone(),
                     cosmwasm_contract.clone(),
-                    stellar::rpc_client::Client::new(
-                        rpc_url.to_string().trim_end_matches('/').into(),
```

Comment:
Are you sure about removing the trimming? Have you tested the negative edge case? I think Haiyi might have added this for a reason